mdgo/core.py

```diff
@@ -1033,8 +1035,15 @@ def get_heat_map(
         """
         nvt_run = self.wrapped_run
         floating_atoms = nvt_run.select_atoms(self.select_dict.get(floating_atom))
-        terminal_atom_type = self.select_dict.get(cluster_terminal)
-        assert terminal_atom_type is not None, "{} not defined in select_dict".format(cluster_terminal)
+        if isinstance(cluster_terminal, str):
+            terminal_atom_type: Union[str, List[str]] = self.select_dict.get(cluster_terminal)
+            assert terminal_atom_type is not None, "{} not defined in select_dict".format(cluster_terminal)
+        else:
+            terminal_atom_type = list()
```

Comment:
Need to reference this variable at line 1063

Reply:
`terminal_atom_type` -> `terminal_atom_types`

Reply:
@rkingsbury Fixed!

Reply:
BTW how do I give you permission to push to this PR?